WcaOnRails/app/models/competition.rb

```diff
@@ -910,6 +910,17 @@ def can_edit_registration_fees?
     true
   end
 
+  def registration_status
+    if registration_not_yet_opened?
+      return "not_yet_opened"
+    elsif registration_past?
+      return "past"
+    elsif registration_full?
+      return "full"
+    end
+    "open"
```

Comment:
If this is moved to `else`, won't you be able to avoid the `return` in the previous if & elsif? (not 100% sure, please check it by running the code)

Reply:
Yep you're right, that would work and be more concise.